tsung/automate_tsung_testing.py

```diff
@@ -5,18 +5,18 @@
 from time import sleep
 import requests
 
-app_instances = ['t3.micro', 't3.medium', 't3.xlarge', 'r5.large', 'r5.4xlarge', 'i3.large']
-db_instances = ['db.t3.micro', 'db.t3.medium', 'db.t3.xlarge', 'db.r5.large', 'db.r5.4xlarge']
-num_instances = [1, 2, 4]
-exclude_instances = {} #{'t3.micro': ['db.t3.micro', 'db.t3.medium', 'db.t3.xlarge']}
+app_instances = ['t3.micro', 'r5.4xlarge']# ['t3.micro', 't3.medium', 't3.xlarge', 'r5.large', 'r5.4xlarge', 'i3.large']
```

Comment:
Double-checking that these are meant to be commented out.

Reply:
Yep, temporarily so that we don't have to run all the tests